ci-operator/config/openshift/cluster-version-operator/openshift-cluster-version-operator-master.yaml

```diff
@@ -31,7 +31,7 @@ build_root:
   image_stream_tag:
     name: release
     namespace: openshift
-    tag: golang-1.13
+    tag: golang-1.15
```

Comment:
I would like to also dump `openshift_release_golang-1.13`, `openshift_release_golang-1.14` and other referenced to older Go versions here.  And probably also `rhel-8-golang-1.15-openshift-4.6`.  Master should just be using `ocp_builder_rhel-8-golang-1.15-openshift-4.7`.